md/sv2-workshop.md

```diff
@@ -217,11 +214,20 @@ cd bitcoin-sv2
 $TP -datadir=$HOME/.bitcoin-sv2-workshop -signet -sv2
 ```
 
+---
+
+## Navigate `mempool.space`
+
+There's a local `mempool.space` block explorer available at:
+
+http://X.Y.Z.W
```

Comment:
I wasn't able to deploy `mempool.space` on the VPS, only locally.

So I will update the slides tomorrow with my local IP on the WiFi hotspot.